frontend/webpack.dev.js

```diff
@@ -1,10 +1,19 @@
+require('dotenv')
```

Comment:
Do we actually need this logic in `webpack.dev.js`? I'm not sure there's a situation where we'd want to build dev assets for prod, unless I've missed something of course.

Reply:
That makes a lot of sense.